csrc/java_evp_keys.cpp

```diff
@@ -328,6 +328,42 @@ JNIEXPORT jbyteArray JNICALL Java_com_amazon_corretto_crypto_provider_EvpEcPriva
     }
 }
 
+/*
+ * Class:     com_amazon_corretto_crypto_provider_EvpEdEcPrivateKey
+ * Method:    getPrivateKey
+ */
+JNIEXPORT jbyteArray JNICALL Java_com_amazon_corretto_crypto_provider_EvpEdEcPrivateKey_getPrivateKey(
+    JNIEnv* pEnv, jclass, jlong keyHandle)
+{
+    jbyteArray result = NULL;
+
+    try {
+        raii_env env(pEnv);
+
+        EVP_PKEY* key = reinterpret_cast<EVP_PKEY*>(keyHandle);
+
+        EVP_PKEY_CTX_auto ctx = EVP_PKEY_CTX_auto::from(EVP_PKEY_CTX_new(key, nullptr));
+        CHECK_OPENSSL(ctx.isInitialized());
+
+        OPENSSL_buffer_auto privateKeyBuffer;
+        PKCS8_PRIV_KEY_INFO_auto pkcs8 = PKCS8_PRIV_KEY_INFO_auto::from(EVP_PKEY2PKCS8(key));
+        CHECK_OPENSSL(pkcs8.isInitialized());
+
+        // This next line allocates memory
+        int bufLen = i2d_PKCS8_PRIV_KEY_INFO(pkcs8, &privateKeyBuffer);
+        CHECK_OPENSSL(bufLen > 0);
+
+        size_t bufLen2 = (size_t)bufLen;
```

Comment:
Never a fan of just adding 2 to a variable name when you can choose something more expressive. Maybe you can go `bufInt` and `bufSize`? Since you have both here, might as well assert equality on the two as well.